### DISCUSSION TITLE
Beta testers search for custom loader

### DISCUSSION THREAD

**Discussion by the-connors** [ps-iowned/ps-iowned#55] [Announcements]
We have custom loader / menu to test. Creating a private beta for testing. We need people to test custom MENU.SYS and report bugs.
- Custom menu probably has many bugs. Detailed bug reports only please.
- We only have NTSC and aliexpress PSIO with firmware 2.5.4. We need more.
- No feature requests, only bug reports on menu.
- May break your PSIO / PS1. Not responsible.

What is custom menu:
- Simple, very fast, custom graphics. Only three buttons on pad.
- Supports booting games from PSIO.
- Supports gameid for memcard pro etc.
- No CDDA/XA yet, but soon.

What is not custom menu:
- No ARM firmware change. Game emulation bugs still there.

If interested, reply to discussion here to get invitation.

**Comment by hammelswade** [comment]
I have an Ali PSIO with cracked FW version 2.6 (build 28) in a PAL 1002 and would like to help you.

**Comment by danhans42** [comment]
I'd like to give it a spin please. See how it compares with the work I've done 👍

**Reply by orabeidm** [reply]
no respect for you, man. you were scared. you don't have the guts.

**Reply by danhans42** [reply]
Read the rules princess. I do not seek your respect, or anyone else's for that matter. There is more to life.

Stick to the task in hand and park your childish insults, perhaps channel the energy into contribution?

**Reply by orabeidm** [reply]
You're an inferior caste. Omitted. You can't talk to. Your words are worthless. An ordinary cowardly chatterbox like a matt kangaroo.

**Reply by the-connors** [reply]
No troll, no flame, no insult. Stop now, or banned. Last warning.

**Reply by danhans42** [reply]
Inferior.. coming from you? You do nothing but post friends screenshots. 

Behave.

**Reply by orabeidm** [reply]
thanks for the access. I sent the information to the guys. they are already working.

**Comment by the-connors** [comment]
Invitations sent. Check emails. Once accept, check https://github.com/ps-iowned/loader-beta-test

**Comment by kimbapslice** [comment]
I can test. Thank you

**Comment by orabeidm** [comment]
I have a little retro game chat, we hacked the original version of the psio firmware. I need access to these files, I will transfer them to my community and get feedback. I will also post it on the Russian forum "4pda.ru " to the appropriate topic. If this helps you, here you can find files for hacking the latest firmware version and installing it on the Chinese version of the hardware. (registration is required on the forum), not in telegram.

https://4pda.to/forum/index.php?showtopic=1053197

https://t.me/+_Gifi6Mn_ldhMjMy

Thx for crack bro!
stupid kangaroo must be eliminated ))))

**Reply by orabeidm** [reply]
Menu System 2.6 (build 28) (16/DECEMBER/2020) aviable

**Comment by orabeidm** [comment]
![photo_2023-03-29_19-32-15](https://github.com/ps-iowned/ps-iowned/assets/60073650/088b9f76-64bf-4dc4-be62-8ae447bee486)

**Comment by hammelswade** [comment]
![20230823_200502](https://github.com/ps-iowned/ps-iowned/assets/119120913/1d120773-f088-40f9-9726-125d80f074c9)

**Comment by longsunzhao** [comment]
I'd be happy to test for you. I have a Cybden PSIO on the latest firmware.

**Comment by johnbaumann** [comment]
Hi there,

I'm interested in testing if there are still any spots available? I have a cart from aliexpress and have a bit of debugging/testing experience on the ps1 + some contributions to a couple open source projects. I can't promise I'll be much help, but seems like a good cause.

**Comment by nicolasnoble** [comment]
Alright, you got me intrigued, I'll bite :)

**Reply by nicolasnoble** [reply]
Works for me: https://youtu.be/KUnZRk2NUzs

**Comment by hammelswade** [comment]
Hammelswade = rippenbiest! 😁

**Comment by KaiohMajin** [comment]
Hello,
I'd be happy to bring some life to my official PSIO paper weight with some DRM free menu testing. :)

**Comment by pkerga** [comment]
Happy to help !

**Comment by Takehaniyasubiko** [comment]
I wish the-connors all the best in hacking PSIO, but until there's an ARM firmware change to address the CD-ROM emulation bugs, we're basically stuck in the same rut.

On the other hand, once the-connors learn how to fix the ARM firmware, Matt is 100% fvcked, and nothing will help him.

It's crazy he STILL doesn't understand the only way to save his business at this point is to release an official FW with fixes ASAP... Oh well, if he just wants to be an utter asshole to the end, let's light a candle for CybDyn, I guess?

**Reply by the-connors** [reply]
Want to yell or want to help with beta? If want to yell, wrong place.

**Reply by Takehaniyasubiko** [reply]
Emphasis is not yelling.

Once you guys start changing the ARM firmware, I'll be happy to test actual games. Not much one can really fix from the menu's end.

**Reply by the-connors** [reply]
Topic is for beta testing of alternative menu. No off topic here pls.

**Reply by Takehaniyasubiko** [reply]
https://github.com/ps-iowned/ps-iowned/assets/2271504/89f95552-eba2-4621-8ec0-417d8805494e

**Comment by Y2K-x** [comment]
I'd love to help test if you'll have me! I have a clone PSIO running 2.6.28 wired up directly to a PU-23. Just curious however, how difficult would it be to go back to the standard firmware? Do you recommend backing up the onboard flash and what not?

**Reply by the-connors** [reply]
No firmware change. Menu is simple game file running off cd emulator.

**Comment by sundeth** [comment]
I have a PSIO installed on a PSOne (PS1 Slim), I can make some tests too.

**Comment by pkerga** [comment]
First test :
- the interface is high in the screen
![PXL_20230825_175516112](https://github.com/ps-iowned/ps-iowned/assets/30766052/432629d5-c05b-436a-83f4-7cbd8805271f)
- Cant load games, seems like rebooting everytime
![PXL_20230825_143725560](https://github.com/ps-iowned/ps-iowned/assets/30766052/f4aa5976-ddd6-432e-9cda-7784eab6529e)

=> screen issue is away on last release 😉

**Comment by LobAnovich** [comment]
I am sure that you are aware of the latest original software hack and can easily test your version of the menu under any conditions.  Moreover, you nowhere indicate that before trying to use your development, you must set certain settings through the original menu, otherwise your software does not perform the declared functions, as well as it does not work on devices with the latest official software.  This is all I can find without participating in testing
